Formula/dbmate.rb

```diff
@@ -18,7 +18,8 @@ class Dbmate < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", "-ldflags", "-s", "-o", bin/"dbmate", "."
+    system "make", "build"
+    bin.install "dist/dbmate"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w"), "-tags", "sqlite_omit_load_extension,sqlite_json"
```

Reply:
Doesn’t it make more sense to use the Makefile so that if flags are changed in the future we keep them in sync?

Reply:
That would be convenient but:
1. the Makefile also does a static build on Linux, which we want to avoid; and,
2. this is more transparent for users who read the formula.

I'm fine with parsing the Makefile and checking to make sure the flags still match and raising an exception if they don't.